src/BeautifulComments/BCBeautifulCommentsSmokeTest.class.st

```diff
@@ -13,3 +13,30 @@ BCBeautifulCommentsSmokeTest >> testRenderDoesNotRaiseError [
 							of: ClyRichTextClassCommentEditorToolMorph ]
 		raise: Error
 ]
+
+{ #category : #tests }
+BCBeautifulCommentsSmokeTest >> testSmokeSignal [
+	BeautifulComments captureErrors: true.
+	Smalltalk globals allClasses do: [ :cl |
+			BeautifulComments  
+							renderComment: cl comment 
+							of: cl
+		
+		 ]	
+]
+
+{ #category : #tests }
+BCBeautifulCommentsSmokeTest >> testSmokeSignalAndChoke [
+	| difficultClasses |
+	"this test disables error handling and tries to render all class comments in the system"
+	"Really, it is a test of the robustness of Microdown parsing and rendering"
+	self skip.
```

Comment:
Is the skip left on purpose?

Reply:
The skip is left on purpose as the errors actually need to be fixed in microdown.